src/main/java/com/payu/artifactory/tools/snapshot/StorageList.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019 PayU
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ *
+ */
+
+package com.payu.artifactory.tools.snapshot;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import lombok.Getter;
+
+@Getter
+@SuppressWarnings("PMD.UnusedPrivateField") // false positive
```

Comment:
?

Reply:
In public super pom we use some old version of pmd, which doesn't proper work with lombok
we can update pmd - but it is other task